hw/amdgpu/bridge/include/amdgpu/bridge/bridge.hpp

```diff
@@ -89,7 +89,9 @@ struct BridgePusher {
     header->vmAddress = address;
     header->vmSize = size;
     std::strncpy(header->vmName, name, sizeof(header->vmName));
-    header->flags |= static_cast<std::uint64_t>(BridgeFlags::VmConfigured);
+    std::uint64_t temp = header->flags;
+    temp |= static_cast<std::uint64_t>(BridgeFlags::VmConfigured);
```

Comment:
No need for temporary variable here, you can just do something like
`header->flags = header->flags | static_cast<std::uint64_t>(BridgeFlags::VmConfigured);`
will need to rewrite it with atomics instead in future

Reply:
@DHrpcs3 done!

I was wondering if this change should also address thread safety, i skip it because i did not want to change any code behavior, just warnings.